### ISSUE TITLE
Chimeric HD Wallets for Shelley

### ISSUE THREAD

**Issue by oldcryptogeek** [Kodex-Data-Systems/Casper#29]
https://github.com/input-output-hk/implementation-decisions/blob/e2d1bed5e617f0907bc5e12cf1c3f3302a4a7c42/text/1852-hd-chimeric.md

**Comment by SebastienGllmt** [comment]
I don't think this document is relevant for the Casper team. The document is meant to explain how wallets should handle Chimeric wallets (wallets that use both the UTXO model and the Account model) but Ethereum doesn't plan to support this. Probably the Casper team is busy enough with their existing feature set and don't want to add another feature that complicated everything.
